src/transformStateWithClones.js

```diff
@@ -1,13 +1,31 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- *
- * @return {Object[]}
- */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const current = { ...state };
+  const resultingArray = [];
+  let newr;
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
Do you really need an index?

Reply:
I may be mistaken, but I though that since we have array of few objects then we should go through this array with loop, to get access to every element, and then compare values of the 'type' key with the commands, we were give

Reply:
use for-of loop instead